napalm_onyx/onyx_ssh.py

```diff
@@ -160,9 +160,9 @@ def close(self):
             self._delete_file(self.backup_file)
         self._netmiko_close()
 
-    def _send_command(self, command):
+    def _send_command(self, command, expect_string=None):
```

Comment:
this was done in another commit
we'll have conflicts here.

Reply:
yes that what I told u about its shared code that all of them will use 

I use it here 

return self.device.send_command(command, expect_string=expect_string)